doc/examples/segmentation/plot_euler_number.py

```diff
@@ -69,15 +69,15 @@
 #
 # In this example, a 3-D cube is generated, then holes and
 # tunnels are added. Euler number is evaluated with 6 and 26 neighborhood
-# configuration. This code is inpired by
+# configuration. This code is inspired by
 # https://matplotlib.org/devdocs/gallery/mplot3d/voxels_numpy_logo.html
 
 
 def make_ax(grid=False):
-    ax = plt.figure().add_subplot(projection='3d')
+    fig, ax = plt.subplots(subplot_kw = {'projection':'3d'})
```

Comment:
I think pre-commit complains about / wants to reformat this line. 

```suggestion
    fig, ax = plt.subplots(subplot_kw={'projection':'3d'})
```

I'm unsure why pre-commit on your side isn't catching that. Maybe your env is outdated somehow? You could try to run
```sh
pre-commit clean
pre-commit install --overwrite --install-hooks
pre-commit run --all
```
and see if that helps.

Reply:
Good catch, @lagru! I'm wondering: Maybe it's just that pre-commit gets triggered upon regular commits, but not upon merge commits.